.github/workflows/build-release.yaml

```diff
@@ -310,11 +310,110 @@ jobs:
             ${{ steps.generate_package.outputs.package_path }}
             ${{ steps.generate_package.outputs.debug_path }}
 
+  build_mac:
+    strategy:
+      matrix:
+        configuration: [FastDebug, Release]
+        compiler: [clang]
```

Comment:
The compiler string must match the options listed in linux/configure_cmake.sh.  There may be another script somewhere that sanitizes "clang" into "clang-9," so it is a good idea to double check.

Reply:
It works fine in PR builds, is overriding `CC` and `CXX` necessary? They default to clang on Mac anyway.

Reply:
According to the notes in configure_cmake.sh the `CC` and `CXX` overrides are a workaround for a bug.  I'm not sure if that's still relevant or not for mac.

Reply:
This works fine on PR checks, and this config is no different, so I guess this doesn't need the workaround?